Library/Homebrew/readall.rb

```diff
@@ -55,7 +58,7 @@ def self.valid_formulae?(tap, bottle_tag: nil)
       next if valid == true || valid&.include?(bottle_tag)
 
       formula_name = file.basename(".rb").to_s
-      formula_contents = file.read(encoding: "UTF-8")
+      formula_contents = File.read(file, encoding: "UTF-8")
```

Comment:
What's the error being fixed here?

Reply:
```
readall.rb:61: Expected Integer but found {encoding: String("UTF-8")} for argument length https://srb.help/7002
    61 |      formula_contents = file.read(encoding: "UTF-8")
                                           ^^^^^^^^^^^^^^^^^
  Expected Integer for argument length of method Pathname#read:
    https://github.com/sorbet/sorbet/tree/9bb99d69aec467149de2bd1d835f6353266f44c2/rbi/stdlib/pathname.rbi#L1061:
    1061 |        length: Integer,
                  ^^^^^^
  Got {encoding: String("UTF-8")} (shape of T::Hash[T.untyped, T.untyped]) originating from:
    readall.rb:61:
    61 |      formula_contents = file.read(encoding: "UTF-8")
                                           ^^^^^^^^^^^^^^^^^
  Detailed explanation:
    Hash does not derive from Integer
Errors: 1
```

Reply:
Thanks, this looks like an upstream bug so will send a PR upstream.